app/service/service-filter.js

```diff
@@ -8,14 +8,14 @@ const serviceFilter = (req, res, next) => {
         })
         .catch(error => {
           if (error.name === 'FetchError') {
-              console.error(error);
+              // console.error(error);
```

Comment:
Similar to the review above, please copy the approach to logging from https://github.com/hmcts/ccd-admin-web/blob/master/src/main/service/service-filter.ts.